joschahenningsen/TUM-Live
# `bug` Delete Chat Replies on Delete

### Motivation and Context
Currently if a message with replies is deleted only the "base" message is deleted, i.e. replies are kept in the database. In other words `deleted_at` is only not-null for base messages. As a consequence, each time we fetch messages deleted "ghost" replies are transmitted also. 

This PR fixes this issue.

### Description
Add `DeleteReplies` to `ChatDao` and call in delete handler.

### Steps for Testing
- 1 Account
- 1 Livestream with the chat activated

1. Log in
2. Navigate to a Livestream
3. Write a message and reply to that message 
4. Delete the base message
5. Use DataGrip (or similar) to check that both the base message and the replies are deleted
